### PR TITLE
run files through ansible-lint

### PR DESCRIPTION
**Pull request by rndmh3ro**
- The single-to-double-quotes comes directly from `ansible-lint --fix` (see the [discussion](https://github.com/ansible-community/ansible-consul/pull/574) from the nomad-role).
- The only part I actively changed is where I moved two tasks to handlers: https://github.com/ansible-community/ansible-nomad/compare/linting?expand=1#diff-3d0ff1709ca48add100327bb2a468e6c508fb92a159c64c4f99ad1df89d9bddeL191

### CODE REVIEW

File: tasks/install_podman.yml [hunk 1]
```diff
@@ -11,17 +11,20 @@
...
+    mode: "0640"
   become: false
   run_once: true
   tags: installation
   when: not nomad_podman_checksum.stat.exists
   delegate_to: 127.0.0.1
 
-- name: Get Nomad Podman package checksum
-  shell: |
+- name: Get Nomad Podman package checksum # noqa no-changed-when
```
Comment (beechesII) [inline tasks/install_podman.yml:26]: Same like `Get Nomad CNI package checksum`

**Review by beechesII — COMMENTED** [review]
Thank you for PR. I added serveral comments.

File: tasks/cni.yml [hunk 2]
```diff
@@ -18,17 +19,20 @@
...
+    mode: "0640"
   become: false
   run_once: true
   tags: installation
   when: not nomad_cni_checksum.stat.exists
   delegate_to: 127.0.0.1
 
-- name: Get Nomad CNI package checksum
-  shell: |
+- name: Get Nomad CNI package checksum # noqa no-changed-when
```
[the remark this answers is not in the document]
Comment (rndmh3ro) [reply tasks/cni.yml:34]: Probably, yes.
However I'd like to leave this for a follow-up ticket.

**Review by beechesII — APPROVED** [review]
lgtm

**Pull request merged**